style.css

```diff
@@ -0,0 +1,1405 @@
+background:transparent;
+border:none;
+font-size:16px;
+
+}
+
+.cards-t1>.wish:hover{
+    transform:scale(1.2);
+}
+.cards-t1>.cart{
+    width:92%;
+    margin-bottom:5px;
+    font-size:14px;
+    border:1px solid rgb(186, 186, 186);
+    color:#282c3f;
+    margin-left:4%;
+    margin-right:4%;
+}
+.cards-t1>.cart:hover{
+    color:black;
+    border:1px solid black;
+    
+}
+.cards-t1>.desc{
+    text-align: center;
+    font-weight: 400;
+    font-size: 14px;
+    color: #535766;
+    margin-bottom: 5px;
+}
+.cards-t1>.price{
+    font-size:14px;
+    font-weight: 500;
+    color:#282c3f;
+    text-align: right;
+    padding-right: 7px;
+}
+
+
+
+/*cards-badge---------------------------------*/
+
+.cards-badge{
+    position:absolute;
+    background-color:#efd3d7;
+    top:15px;
+    width:50px;
+    text-align:center;
+    text-overflow:ellipsis;
+    box-shadow: 4px 4px 6px #686868b3;
+    border-top-right-radius:3px ;
+    border-bottom-right-radius:3px ;
+}
+
+
+
+
+/*cards dismiss*/
+
+.cards-dismiss{
+    position:absolute;
+    top:5px;
+    right:5px;
+    background-color:rgb(216, 216, 216);
+    border-radius:50%;
+    outline:none;
+    width:25px;
+    height:25px;
+    border:none;
+    border:1px solid rgb(216, 216, 216);
+    transition:border 0.5s ease;
+    cursor:pointer;
+}
+.cards-dismiss:hover{
+  border:1px solid gray;
+}
+.cards-dismiss:active{
+    transform:scale(0.9);
+}
+
+
+
+
+
+/*responsive image*/
+
+.responsive-img{
+   
+    min-width:200px;
+}
+.responsive-img>img{
+    width:100%;
+}
+
+.responsive-img.sm{
+    max-width:20%;
+}
+.responsive-img.md{
+    max-width:50%;
+}
+.responsive-img.lg{
+    max-width:80%;
+}
+
+
+
+/*round image*/
+
+.round-img{
+    max-width:50%;
+    min-width: 200px;
+}
+.round-img>img{
+    width:100%;
+    border-radius:50%;
+}
+
+.round-img.sm{
+    max-width:20%;
+}
+.round-img.md{
+    max-width:30%;
+}
+.round-img.lg{
+    max-width:40%;
+}
+
+
+
+/*input text*/
+.input{
+    position:relative;
+   
+    padding-top:20px;
+   
+   
+    
+}
+.input .input-text{
+    border:none;
+    appearance:none;
+    -webkit-appearance: none;
+    -ms-appearance:none;
+    -moz-apperance:none;
+    background:#f3f4f6;
+    padding:12px;
+    border-radius: 5px;
+    width:50%;
+    outline:none;
+    font-size:inherit;
+    box-shadow: none;
+    margin:1em 0;
+    height:18px;
+}
+
+.input.sm>.input-text{
+    width:20%;
+    height:0.5em;
+}
+
+.input.md>.input-text{
+    width:30%;
+    height:0.7em;
+}
+.input.lg>.input-text{
+    width:40%;
+    height:0.9em;
+}
+
+
+
+.input .placeholder{
+    position:absolute;
+    left:12px;
+    top:calc(50% + 10px);
+    transform:translateY(-50%);
+    color:#aaa;
+    transition:top 0.2s ease,
+    font-size 0.2s ease,
+    color 0.2s ease;
+}
+.input .input-text:focus + .placeholder{
+    top:0px;
+    font-size:12px;
+    color:#222;
+}
+.input .input-text:valid + .placeholder{
+    top:0px;
+    font-size:12px;
+    color:#222;
+}
```

Comment:
This code is repetitive, use ``` , ``` to separate the class names and write the same properties